x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/job_creator.ts

```diff
@@ -327,11 +328,18 @@ export class JobCreator {
 
   public set modelMemoryLimit(mml: string | null) {
     if (mml !== null) {
-      this._job_config.analysis_limits = {
-        model_memory_limit: mml,
-      };
+      if (this._job_config.analysis_limits === undefined) {
+        this._job_config.analysis_limits = {};
+      }
```

Comment:
```suggestion
    this._job_config.analysis_limits ??= {};
```

Reply:
babel doesn't like nullish coalescing assignment, it can only be used server side.
I believe core would have to add the plugin to support it.